File: android/app/src/main/java/com/agentremote/app/AppConfig.kt
package com.agentremote.app

object AppConfig {
    val GATEWAY_HOST = BuildConfig.GATEWAY_HOST
    val GATEWAY_ORIGIN = BuildConfig.GATEWAY_ORIGIN
    val ENROLL_REQUEST_URL = "$GATEWAY_ORIGIN/__kimi_enroll/request"
    val ENROLL_STATUS_URL = "$GATEWAY_ORIGIN/__kimi_enroll/status"
    val APPS_URL = "$GATEWAY_ORIGIN/__agent_remote/apps"
    val CONTROL_TOKEN = BuildConfig.CONTROL_TOKEN
    val BOOTSTRAP_PASSWORD = BuildConfig.BOOTSTRAP_PASSWORD
    val BOOTSTRAP_FINGERPRINT = BuildConfig.BOOTSTRAP_FINGERPRINT

    fun appActionUrl(id: String, action: String) = "$APPS_URL/$id/$action"
}

File: ios/AgentRemote/AppConfig.swift
import Foundation

enum AppConfig {
    static let gatewayHost = "__GATEWAY_HOST__"
    static let gatewayOrigin = "__GATEWAY_ORIGIN__"
    static let enrollRequestURL = URL(string: gatewayOrigin + "/__kimi_enroll/request")!
    static let enrollStatusURL = URL(string: gatewayOrigin + "/__kimi_enroll/status")!
    static let appsURL = URL(string: gatewayOrigin + "/__agent_remote/apps")!
    static let controlToken = "__CONTROL_TOKEN__"
    static let bootstrapPassword = "__BOOTSTRAP_PASSWORD__"
    static let bootstrapFingerprint = "__BOOTSTRAP_FINGERPRINT__"

    static func appActionURL(_ id: String, _ action: String) -> URL {
        URL(string: gatewayOrigin + "/__agent_remote/apps/\(id)/\(action)")!
    }
}
